src/BundleSource.ts

```diff
@@ -224,7 +224,7 @@ ${file.headerContent ? file.headerContent.join("\n") : ""}`);
             this.concat.add(null, "(FuseBox)");
         }
         if (this.context.sourceMapsProject || this.context.sourceMapsVendor) {
-            let sourceName = /[^\/]*$/.exec(this.context.bundle.name)[0];
+            let sourceName = this.context.output.filename;
```

Comment:
Here i see a problem.
If you do
```
bundle("foo/app")
```

Sourcemaps will be referencing a folder + filename (foo/app.js), instead of a relative path (app.js)
Please, test that